test/integration/02-user-wallet/02-index.spec.ts

```diff
@@ -36,7 +36,9 @@ describe("UserWallet", () => {
     expect(userWallet2.user.oldEnoughForWithdrawal).toBeFalsy()
 
     // in 6 days:
-    const date = Date.now() + yamlConfig.limits.oldEnoughForWithdrawal - 60 * 60 * 1000
+    const genericLimits = getGenericLimits()
+    const date =
+      Date.now() + genericLimits.oldEnoughForWithdrawalMicroseconds - 60 * 60 * 1000
```

Comment:
I know this magic number wasn't added in this PR, but we could have it a const and use it here and in #L55

Reply:
**_Done: https://github.com/GaloyMoney/galoy/pull/393/commits/1d926d4e534f4a0478158b683b28db92cdb6d773_**